src/features/contacts/view/index.tsx

```diff
@@ -14,7 +14,9 @@ import { BreadCrumbNavigation } from "components/breadcrumbs";
 export const ContactBase = () => {
   const { state } = useLocation();
   const contact = state as ContactDetailsDto;
-  const contactFullName = `${contact.firstName} ${contact.lastName}`;
+  const contactFullName = `${contact.firstName ? contact.firstName : ""} ${
```

Comment:
as before you can make it easier like
contact.firstName || "

Reply:
Fixed it.